src/python/pants/backend/python/subsystems/poetry.py

```diff
@@ -24,7 +24,7 @@ class PoetrySubsystem(PythonToolRequirementsBase):
     options_scope = "poetry"
     help = "Used to generate lockfiles for third-party Python dependencies."
 
-    default_version = "poetry==1.1.8"
+    default_version = "poetry==1.1.14"
```

Comment:
Interestingly, we have no lockfile for the Poetry tool! I will not tackle that in this hotfix PR though.

Reply:
Poetry tool lockfile: https://github.com/pantsbuild/pants/pull/16117